tests/k8s/cluster/cluster-manager.bash

```diff
@@ -193,16 +196,50 @@ function install_kubeadm() {
 }
 
 function install_cilium_config(){
-    sudo mkdir -p /var/lib/cilium
+    cat > "${dir}/cilium-config.yaml" <<EOF
```

Comment:
As discussed offline, let's use the template and then `sed` to adjust it so we test the yam template that we provide.

Reply:
Fixed